e2e/pvc_test.go

```diff
@@ -24,6 +21,9 @@ var pvcTestYAML string
 var pvcApplyYAML string
 
 var _ = Context("pvc_test", func() {
+	if doUpgrade {
+		return
+	}
```

Comment:
This scenario was not originally required to be run in the upgrade test, so we have changed it to skip the entire test.

As in the following test:
https://github.com/cybozu-go/moco/blob/b63186fc492f206813e1684abaecbbc0b4ff8c1e/e2e/lifecycle_test.go#L24